src/libprojectM/ProjectM.cpp

```diff
@@ -463,7 +461,7 @@ void ProjectM::Reset()
     ResetEngine();
 }
 
-void ProjectM::Initialize(int gx, int gy, int fps, int texsize, int width, int height)
+void ProjectM::Initialize(int meshResolutionX, int meshResolutionY, int targetFps, int width, int height)
```

Comment:
We should be on the lookout for the letters x, y and z (and also rgba). Most of the time, these can be replaced by `glm::ivec2`, `glm::vec3` or `glm::vec4`.

Reply:
Would definitely make the parameter list easier to read, yes.

Reply:
I removed the parameters, at least in the `ProjectM` class. They're just settings vars.